src/app/index.js

```diff
@@ -140,15 +139,26 @@ export default yeoman.generators.Base.extend({
 
     let defaults = {
       bower: configs.hasOwnProperty('bower') ? !!configs.bower : true,
-      changelog: configs.hasOwnProperty('changelog') ? !!configs.changelog : true,
-      docs: configs.hasOwnProperty('docs') ? !!configs.docs : false,
       ghooks: configs.hasOwnProperty('ghooks') ? !!configs.ghooks : 'lint',
-      lang: configs.hasOwnProperty('lang') ? !!configs.lang : false,
       license: this._licenseDefault,
-      sass: configs.hasOwnProperty('sass') ? configs.sass : false,
+
+      noShimVideojs: configs.hasOwnProperty('noShimVideojs') ? !!configs.noShimVideojs : false,
```

Comment:
Yeah, seems a bit too fiddly and I worry that users will get themselves into trouble.